### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by iiAzui**

### CODE REVIEW

File: wizard.gd [hunk 1]
```diff
@@ -5,7 +5,7 @@ extends CharacterBody2D
 @export var JUMP_VELOCITY = -350.0 ## Jump height, kinda.
 @export var rampin = 5 ## How fast to go towards full speed.
 @export var floaty: float = 150 ## How far the whizard can jump.
-@export var feather: int = 20## How fast the wizard should stop after they stop jumping.
+@export var feather: int = 20## How fast the whizard should stop after they stop jumping.
```
Comment (lishaduck) [inline wizard.gd:8]: This feels really nitpicky, but there _should_ be a space. No matter.

Comment (iiAzui) [reply wizard.gd:8]: But you didn't add a space you added a h to wizard making it spelled incorrectly???

Comment (lishaduck) [reply wizard.gd:8]: The _h_ is because the wizards are called (Whiz)ards. I must have just forgotten to add a space when I suggested the fix.

Comment (iiAzui) [reply wizard.gd:8]: When did we call them whizards

Comment (lishaduck) [reply wizard.gd:8]: I suggested it yesterday, and Matthew seemed to approve.

**Pull request merged**